sklearn/covariance/_graph_lasso.py

```diff
@@ -188,7 +188,7 @@ def graphical_lasso(
     GraphicalLassoCV : Sparse inverse covariance with
         cross-validated choice of the l1 penalty.
 
-    Notes
+  Notes
```

Comment:
This does not need to change. It is causing [CI linting failures](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=38001&view=logs&j=32e2e1bb-a28f-5b18-6cfc-3f01273f5609&t=fc67071d-c3d4-58b8-d38e-cafc0d3c731a) and will later result in `numpydoc` issues.